### DISCUSSION TITLE
Discussion on a solution for outdated page link titles (which cause confusion)

### DISCUSSION THREAD

**Discussion by ReessKennedy** [outline/outline#4346] [Ideas]
## Issue
* Many inline links to internal pages are created
* Page titles are often changed/updated
* Those original links now become confusing -- they may still link to working content but since the title has changed, confusion sets in. 

Example of two links I was just confused about that end up pointing to same article 
<img width="545" alt="Screen Shot 2022-10-26 at 3 31 46 PM" src="https://user-images.githubusercontent.com/3970267/198119871-aec474fc-d41d-41c7-8fbd-beb678fb82d1.png">

## Solution options
* A little refresh icon on the link in question that will refresh the title of that link if you click on it <- I think this option is best
* One button that will update page titles for all links in a page <-- Also a good additional option, maybe, when you have a huge number of links on a page and know you can safely update them
* Automatically just keep these page titles locked in sync <-- Probably not a good idea as it's sometimes nice to be able to customize the text to a page and Outline currently does allow for this

**Comment by tommoor** [comment]
These backlinks used to update automatically, this functionality was lost with the move to collaborative editing – but could be restored with some effort

**Reply by ReessKennedy** [reply]
Yeah, this might be a good community discussion. As outlined above, perhaps automatic updates cause problems but a voluntary "link refresh" either at the page level or link level, in the popup, would be safe and save time and sometimes prevent confusion base don my current usage.

**Reply by TLRS-Sandwich** [reply]
I am in favour of having this ability. As the person who's responsible for creating and maintaining both our customer knowledge base and our internal knowledge base, I sometimes find myself creating pages that end up needing to be re-titled for the following reasons such as:
* I took the information from two pages and squished them into one.
* My boss decided that he didn't like the name.
* I moved something from being customer-facing to being internal-facing.
What would be the detriment to incorporating this ability again?

**Reply by tommoor** [reply]
This was solved quite a while ago by using the @mention menu to insert documents.

The titles and icons will automatically stay up-to-date.

**Reply by TLRS-Sandwich** [reply]
So if you use the @mention menu in the body of a page, it won't also populate the Backlinks section at the bottom?
